configs/components/echam/echam.yaml

```diff
@@ -496,6 +492,46 @@ choose_yr_perp_off:
                                 runctl:
                                         l_orbsvop87: ".false."
 
+choose_with_lco2_emis:
+    True:
+        # Run "emission-driven" echam experiment (in cmip6 context e.g. `esm-piControl` or `esm-hist`) where
+        # atmosheric CO2 concentration is not prescribed but calculated by the model setup.
+        # CO2 emissions need to be provided as forcing file (`carbon_emissions.nc`).
+        add_namelist_changes:
+            namelist.echam:
+                submodelctl:
+                    lco2: true # co2 submodel interacting with jsbach; true for emission-driven; if true, lco2_scenario will be set false during model run
+                radctl:
+                    ico2: 1 # 1 = interactively calculated co2 for emission-driven (lco2=true)
+                co2ctl:
+                    lco2_emis: true # read carbon emissions; need carbon_emissions.nc in work
+                    lco2_2perc: true # limit maximum co2 growth rate to 2% wrt previous time step
+        add_choose_scenario: # other loop order (1:add_forcing_files 2:add_choose_scenario) does not work 
+            HIST:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_hist
+            ssp126:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_ssp126
+            ssp245:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_ssp245
+            ssp370:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_ssp370
+            ssp534os:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_ssp534os
+            ssp585:
+                add_forcing_files:
+                    carbon_emissions: carbon_emissions_ssp585
```

Comment:
All this repetition can be removed once `feat/ssp_generalization` is merged here.

Reply:
Same thing, we added a lot of repetition. After the rework from Sebastian, many of the repetition in ECHAM where gone. Now they are slowly creeping in again.